services/web-app/pages/contribute/bugs-and-requests.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
+  const seo = {
+    title: 'Bugs and requests'
+  }
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <RemoteMdxLoader source={source} />
+    </>
+  )
+}
+
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/contributing/bugs-and-requests.mdx'
```

Comment:
<img width="865" alt="image" src="https://user-images.githubusercontent.com/40550942/172720538-175ef343-4f48-42e7-ac4a-c733f3253be3.png">
takes me to 404 (/help/contact-us)